server/core/config/raw/repo_cfg_test.go

```diff
@@ -265,8 +265,8 @@ func TestConfig_ToValid(t *testing.T) {
 			},
 			exp: valid.RepoCfg{
 				Version:                    2,
-				Automerge:                  false,
-				ParallelApply:              false,
+				Automerge:                  nil,
```

Comment:
It's nil, since we have to account for a ternary condition. In order to determine whether to default to the global value, we need to know that the value was not set in the repoconfig. The struct param is defined as a pointer in order to satisfy this need.